static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
i just tested this and here are the results.

When lookups `q` query is set to TRUE:
1) Pass in q as a string (Node, IP, Mac) and the returned result is the expected array
2) Pass in `nil`, then and error comes back with `cannot use nil as type string in field value`
3) Pass in `""` (completely blank), the returned result is *everything* in an array as expected

When lookups `q` query is set to FALSE:
1) Pass in q as a string (Node, IP, Mac) and the returned result is the error: `cannot use "56dde3441722c192796e3a38" (type string) as type *string in field value`
2) Pass in `nil`,  the returned result is *everything* in an array as expected